examples/ct/handlers_test.go

```diff
@@ -163,7 +163,6 @@ func TestPostHandlersOnlyAcceptPost(t *testing.T) {
 	// Anything in the post handler list should only accept POST
 	for _, hp := range allPostHandlersForTest(client) {
 		s := httptest.NewServer(hp.handler)
-		defer s.Close()
```

Comment:
What was the complaint for this one (and similar)?  The `defer` variant seems more robust to me.